Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM maven:3.6.3-jdk-13 as build_jar
+RUN yum install git -y
+WORKDIR /jar
+COPY pom.xml .
+RUN mvn dependency:go-offline
+COPY src/ /jar/src/
+RUN mvn package
```

Comment:
@Sammers21 I would recommend changing(just single RUN) the above instructions a little bit just to one layers:
```
FROM maven:3.6.3-jdk-13 as build_jar
WORKDIR /jar
COPY pom.xml .
COPY src/ /jar/src/
RUN yum install git -y && \ 
        mvn dependency:go-offline && \
        mvn package
```

Reply:
Since we use a 2-staged build, the number of layers in the first step image does not important.

Reply:
@Sammers21 ok